README.md

```diff
@@ -1,7 +1,9 @@
 # Documentation
 
-This repository provides a bash script for building
-[Codethink's GCC](https://github.com/CodethinkLabs/gcc)
-with [Omnibus](https://github.com/CodethinkLabs/omnibus-codethink-toolchain)
-(gcc_omnibus_build_script.sh).
-- Example usage: `./gcc_omnibus_build_script -g "fortran-extra-legacy-7.2"`.
+This repository provides:
+1. bash script for building [Codethink's GCC](https://github.com/CodethinkLabs/gcc)
+   with [Omnibus](https://github.com/CodethinkLabs/omnibus-codethink-toolchain)
+   (gcc_omnibus_build_script.sh).
+    - Example usage: `./gcc_omnibus_build_script -g "fortran-extra-legacy-7.2"`.
+2. bash script for running available tests suites for Codethink's GCC
+   (gcc_test_script.sh).
```

Comment:
this is technically a shell script not a bash script, given that you have used #!/bin/sh instead of #!/bin/bash